File: padroesCriacionais/padrao-de-projeto-abstractFactory/php/tests/unit/HardMapTest.php
<?php

namespace Abstractfactory\tests\unit;

use Abstractfactory\Php\HardMap;
use PHPUnit\Framework\TestCase;

class HardMapTest extends TestCase
{
    public function testShouldReturnHardMap(): void
    {
        $hardMap = new HardMap();
        $this->assertEquals("gera mapa no modo dificil", $hardMap->makeMap());
    }
}

File: padroesCriacionais/padrao-de-projeto-abstractFactory/java/src/test/java/HardMapTest.java
import abstractFactory.HardMap;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class HardMapTest {

    @Test
    public void shouldReturnHardMap() {
        HardMap hardMap = new HardMap();
        assertEquals("gera mapa no modo dificil", hardMap.makeMap());
    }
}
